src/convertToObject.js

```diff
@@ -7,6 +7,21 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+
+  const stylesObject = {};
+
+  const lines = sourceString.split(';');
+
+  lines.forEach(function (line) {
+    const part = line.split(':');
+
+    if (part.length >= 2) {
```

Comment:
This condition allows empty property names if there is whitespace before the colon. You might want to ensure that the property name is not empty after trimming.